frontend/src/index.js

```diff
@@ -2,37 +2,31 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import "bootstrap/dist/js/bootstrap.bundle.min.js";
 
 import { createRoot } from "react-dom/client";
-import App from "./App";
+import ReportDataHandler from "./ReportDataHandler";
 import { createHashRouter, RouterProvider } from "react-router-dom";
+import DragAndDrop from "./components/DragAndDrop/DragAndDrop";
 
 const router = createHashRouter([
   {
     path: "/",
```

Comment:
I would say root path should still point to newest draft version. Updating local report is rather unusual action for the user, but I might be wrong 😄

Reply:
Perhaps coming around to this as well, so sorry for misleading you a bit @AgniveshChaubey -- let's maybe go with this still then (redirecting to latest), and we can think about how we "automatically" make it a bit more prominent when someone's using the UI offline (i.e. locally) later.

Reply:
Yes, I agree with Harrel here too!

Reply:
> If I am not wrong, you mean rendering the upload report component at / route when users are using UI locally?
However, the option to upload a report will still be there in navbar's dropdown list.

Yes -- in other words if we detect that the base URL is `https://localhost` (or the reverse, not `https://bowtie-...`) then maybe we skip showing any dialects and only show the local upload. This will probably be more obvious as part of #296 because the obvious way to implement that I think will *force* us to do something different for the local case.

Reply:
Actually an even better way of doing the above might just be trying to fetch the dialect file and whenever it fails falling back to showing the "upload report" widget.

Reply:
I think checking for `localhost` would be good option because the fetching process is being done in loader function and the fetching URL itself is dependent on route `params` and also its corresponding component is dependent on the status of loader function!

Reply:
But ya, if you want that way, we still can achieve that by redirecting to `upload report` component from the loader function itself if it fails fetching the data. But in that case we'll have to compromise with the URL (as currently `<ReportDatahandler />` is being rendered at `/` route)